### PR TITLE
Developer manual concept creation

### LINKED ISSUE #872: Developer manual concept creation
Describe how to use concept wizard to create a new concept.

### PR DESCRIPTION
**Pull request by andriikovalov-dlr**
Closes #872

### CODE REVIEW

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc [hunk 1]
```diff
@@ -158,6 +158,76 @@ Congratulations. You have successfully set up your Git integration!
...
+
+[%header,cols=">8,62a,40a"] 
+|===
+
+|Step
+|Task
+|Image
+
+|{counter:step-concept}.
+|Start with clicking menu:File[New > Other...] or kbd:[CTRL+N]. Then from the _new_ dialog select menu:VirSat Model Extension Concept[VirSat Model Extension Concept Wizard] and press btn:[Next>].
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc:171]: 1 sentence per line.

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc [hunk 2]
```diff
@@ -158,6 +158,76 @@ Congratulations. You have successfully set up your Git integration!
...
+The purpose of the migrator is to migrate data modeled with the older concept version to the new version.
+For most changes the auto-generated migrator is sufficient, however if some custom migration logic is required, it can be added to the migrator class.
+
+==== UI plugin
+
+Auto-generated UI plugin contains UI snippets, commands and handlers that extend Virtual Satellite UI for this concept.
+
+==== Test plugin
+
+Test plugin contains some auto-generated unit tests. You can add more unit tests as you develop your concept.
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc:224]: 1 sentence per line.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Great! Only a super minor thing. We try to have only one sentence in a line in the asciidoc source.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by franzTobiasDLR — APPROVED** [review]

**Pull request merged**